lib/event_manager.rb

```diff
@@ -1,9 +1,8 @@
+require "csv"
 puts "EventManager Initialized!"
 
-lines = File.readlines "event_attendees.csv"
-lines.each_with_index do |line, index|
-  next if index == 0
-  columns = line.split(",")
-  name = columns[2]
+contents = CSV.open "event_attendees.csv", headers: true, header_converters: :symbol
```

Comment:
One thing I would say - the Ruby community sometimes uses `( )` around items, and other times not.

Personally - for readability - I would say use them.

Ex: `CSV.open("event_attendees.csv", headers: true, header_converters: :symbol)`

@courtsimas may not agree with me though.